### ISSUE TITLE
Suggestion: Add PersistentKeepalive Parametr for client Generation

### ISSUE THREAD

**Issue by Andr0s91** [perara/wg-manager#44]
Can you please add, on client configuration file download, the PersistentKeepalive parameter? I think this is the only missing parameter at this time!
thank you!

**Comment by tony1661** [comment]
I'd like this too. Let me see if I can implement it this week.

**Comment by tony1661** [comment]
@Andr0s91 I verified that this functionality has been implemented already. It seems that the Docker Hub repo is a bit behind. Possibly we can get an updated build on Docker Hub? @perara :)

**Comment by perara** [comment]
Yes! Ill try to do some work on this in a couple of days. I made a few issues earlier where specificslly the ci part is tightly coupled with this.

Nice holidays!
